helm/policies-common/tests/abs/test_common_default.py

```diff
@@ -55,3 +55,18 @@ def test_kubeadmconfig_policy(kubeadmconfig) -> None:
     assert kubeadmconfig['metadata']['labels']['cluster.x-k8s.io/watch-filter'] == ensure.watch_label
     assert kubeadmconfig['spec']['joinConfiguration']['nodeRegistration']['kubeletExtraArgs']['healthz-bind-address'] == "0.0.0.0"
     assert kubeadmconfig['spec']['joinConfiguration']['nodeRegistration']['kubeletExtraArgs']['node-labels'] == "role=worker"
+
+@pytest.mark.smoke
+def test_silence_heartbeat_policy(silence) -> None:
```

Comment:
You need `from ensure import silence` at the top of the file

Reply:
Thanks :) Now I just need the new apiextentions :D

Reply:
This is the line you need to change:
https://github.com/giantswarm/kyverno-policies/blob/23cc4ffff1f1b52e99c177ec627f56231f95ea6a/hack/setup-local.sh#L5

Reply:
I also wrote a local testing guide if you want to try that:
https://github.com/giantswarm/kyverno-policies#development

Reply:
Yep I know but I want it to be merged first :D

Reply:
Thanks buddy <3 You are really helpful :D